src/drivers/qtouch/qtouch.c

```diff
@@ -273,6 +273,29 @@ static touch_ret_t touch_sensors_config(void)
 	return (touch_ret);
 }
 
+
+/*
```

Comment:
move documentation to the header file

Reply:
This is not consistent with the rest of `qtouch.c/.h`

Reply:
would be inconsistent with `qtouch.c/.h`